src/components/ProjectsList.astro

```diff
@@ -30,7 +30,11 @@ for (const project of projects) {
 {projects.map(async (project) => {
   return (
     <section>
-      <header>{project.title}</header>
+      {project.link ? (
+        <a href={project.link}><header>{project.title}</header></a>
+      ) : (
+        <header>{project.title}</header>
+      )}
       {project.video ? (
         <video src={project.videoPath} loop muted />
```

Comment:
What do you think about just adding `#t=8` to the end of the video `src` attribs?
```
<video src={`${project.videoPath}#t=8`} loop muted />
```

8 seconds actually seems to work pretty well for all the videos available at this point, but it could of course be configurable in the YAML.  Starting to get a bit OTT?

Reply:
Starting 8 seconds in would be fine with me for now. I agree that a lot more refinement could be done with the videos, especially the newer ones, but maybe this is good enough for now.

Reply:
I'll just do this for now, then, and we'll iterate from there as inclination, inspiration, and time allow :)